Makefile

```diff
@@ -162,6 +162,12 @@ test-coverage:  ## Run test coverage.
 	$(PYTHON) -m coverage html
 	$(PYTHON) -m webbrowser -t htmlcov/index.html
 
+test-ci: install-sysdeps
```

Comment:
For consistency with the rest of the file, could you please rephrase it as:

```Makefile
test-ci: 
     {MAKE} install-sysdeps
```

Reply:
done